### DISCUSSION TITLE
Reason for distance to be limited to 1000 mm ?

### DISCUSSION THREAD

**Discussion by tgirod** [dune3d/dune3d#125] [General]
What is the reason for that limitation? Is it a technical one? Is it arbitrary?

**Comment by carrotIndustries** [comment]
This has been fixed in https://github.com/dune3d/dune3d/commit/a7df8fd18fe98ac03fa143c4de1c0d24330257f8

Was a purely arbitrary restriction.

**Reply by ruevs** [reply]
@carrotIndustries keep in mind that the solver has trouble converging with "large objects. It would be interesting if the problem is the same in Dune3D - I guess it would be - but am not on a PC now to try. Since you (probably) are the person who most recently touched the SolveSpace solver perhaps you can shed some light on the problem.

For cross reference and examples:
https://github.com/solvespace/solvespace/issues/1466#issuecomment-2173545087
https://github.com/solvespace/solvespace/issues/246
https://github.com/solvespace/solvespace/issues/1105

This is the simplest model causing the problem that I know of:
https://github.com/solvespace/solvespace/issues/1354
